.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: "New Version Actions"
+
+on:
+  push:
+    branches: # only trigger when a commit was pushed to main
+      - main
```

Comment:
I realized the main branch of this repo is called `master`, probably since it's a old repo, so we should update this part.
I believe for eslint repos, it's `main`